### DISCUSSION TITLE
Adding reddit titles to bookmarks

### DISCUSSION THREAD

**Discussion by Gilgeam** [floccusaddon/floccus#1778] [Q&A] [answered]
Hey everyone,

I just discovered floccus and got really excited about finally having a reliable bookmark tool that stores data on my own terms. 

I'm running into a bit of trouble, however, since a big part of stuff I'd like to preserve are reddit links. If I share to floccus on my android, floccus will take the URL, but will generate the generic reddit tagline such as 'Reddit - Dive into anything', while having just name of the thread would be a lot more usable. Or rather, usable at all. Raindrop automatically pulls that info, so I imagine it's not impossible to do. Maybe I'm just missing something.

Does anyone have a piece of advice how to get that done? That'd be awesome!

**Comment by marcelklehr** [comment]
This is similar to a bug report we have about the YouTube app. I guess we can just always fetch the title automatically, and never rely on the shared title. I'll look into this.

**Reply by Gilgeam** [reply]
I'd absolutely appreciate it! I'm a noob at coding and I always, always end up underestimating how much work stuff like this can be even though they don't seem so complicated at first.

Thank you for at least consider it, that's more than I can ask!

**Comment by marcelklehr** [comment, marked as answer]
I found the issue. The user agent was to blame for the wrong title. With a neutral UA the title is obtained correctly 🎉
